doc/source/tune/examples/index.rst

```diff
@@ -4,6 +4,16 @@
 Ray Tune Examples
 =================
 
+.. toctree::
+    :hidden:
+
+    ml-frameworks
+    experiment-tracking
+    hpo-frameworks
+    Other Examples <other-examples>
+    Exercises <exercises>
```

Comment:
I'm not sure why this is happening but it looks like the sub-toctrees are getting generated again at this level

![image](https://github.com/ray-project/ray/assets/3967392/74f762a4-a4e8-4504-b713-804c27e0ce20)

Reply:
Good find. I think this is happening because `tune/examples/index.rst` has a toctree with references to a bunch of pages. Further down, we _literally include those pages_. Each of these includes its own toctree as well, which gets concatenated onto the sidebar nav tree, which is built from the toctree. 

I'm really not sure why we are doing this. IMO we should change it; I'd vote to just remove the `include` directives, and make these pages completely separate. How would you like to resolve this? @matthewdeng

Reply:
I think removing the includes in this `index.rst` is a good solution. Could we remove the toctree in `_toc.yaml` and create a `:toctree:` inside `tune/examples/index.rst` instead?

Reply:
> Could we remove the toctree in _toc.yaml and create a :toctree: inside tune/examples/index.rst instead?

Yes, these were part of the core changes made in this PR :tada: !